contracts/authorization/src/contract.rs

```diff
@@ -693,12 +704,47 @@ fn process_processor_callback(
         ));
     }
 
-    // Updated the result
+    // Update the result
     callback.execution_result = execution_result;
-
     PROCESSOR_CALLBACKS.save(deps.storage, execution_id, &callback)?;
 
+    // Reduce the current executions for the label
+    CURRENT_EXECUTIONS.update(
+        deps.storage,
+        callback.label.clone(),
+        |current| -> StdResult<_> { Ok(current.unwrap_or_default().saturating_sub(1)) },
+    )?;
+
+    // Check if a token was sent to perform this operation and that it wasn't started by the owner
+    let authorization = AUTHORIZATIONS.load(deps.storage, callback.label.clone())?;
+    let mut messages = vec![];
+    if let OperationInitiator::User(initiator_addr) = &callback.initiator {
+        if let AuthorizationMode::Permissioned(PermissionType::WithCallLimit(_)) =
```

Comment:
Can you explain more about user vs owner here? 
Owners basically are free to do any operation for free?

Reply:
Yes, Owner can insert / remove actions from the any queue for free without restrictions